.github/CONTRIBUTING.md

```diff
@@ -96,7 +96,19 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirity.
+
+Any pull-request that does not adequately complete the pull-request template will not be merged and may be closed by maintainers (It is up to the discretion of the maintainers to decide if the PR will be accepted, closed or marked as 'do not merge').
+ - Balance changes must indicate that they affect the balance of the game.
+ - Changes must be documented in their entirity, and the full extend of their effects mentioned. (For example, if you change it so all mobs are half speed, don't label the PR as 'monkeys now move twice as slow'). Failing to document the full extent of the changes may be seen as malicious and could result in a repo-ban in a worst case scenario.
```

Comment:
This sounds a bit harsh i am not saying you should not document your changes but saying you can get repo banned just because you didn't properly write down a small change is a bit much also this doesn't really specifiy where you should document those changes after you made the PR(i personally like to put them in commit messages and if its major enough mention them in the about section) neither does it say how well you actually should document them it just provides a bad example no good example.

Reply:
Thinking about it i may have interpreted this a bit to harsh myself but mentioning where to document and what would count as a good example at least sounds like a good idea

Reply:
Something like "repeated violation" would be more appropriate here.

Reply:
It does say 'could' and that its a worse case scenario rather than you will be banned for this, but I can update it.